content/en/docs/tasks/observability/distributed-tracing/opentelemetry/index.md

```diff
@@ -0,0 +1,181 @@
+for both scenarios.
+
+## Deploy OpenTelemetry Collector
+
+1.  Create a namespace for the Collector:
+
+{{< text bash >}}
+$ kubectl create namespace otel-collector
+{{< /text >}}
+
+1.  Deploy the OpenTelemetry Collector. You can use this example configuration as an starting point:
+[`otel.yaml`]({{< github_blob >}}/samples/open-telemetry/otel.yaml)
+
+{{< text bash >}}
+$ kubectl -n otel-collector apply -f otel.yaml
+{{< /text >}}
+
+## Installation
+
+All tracing options can be configured globally via `MeshConfig`.
+To simplify configuration, it is recommended to create a single YAML file
+which you can pass to the `istioctl install -f` command.
+
+### Exporting via gRPC
+
+In this example, traces will be exported via OTLP/gRPC to the OpenTelemetry Collector.
+
+{{< text bash >}}
+$ cat <<'EOF' > tracing.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    extensionProviders:
+    - name: otel-tracing
+      opentelemetry:
+        port: 4317
+        service: opentelemetry-collector.otel-collector.svc.cluster.local
+EOF
+$ istioctl install -f ./tracing.yaml --skip-confirmation
+kubectl label namespace default istio-injection=enabled
+{{< /text >}}
+
+### Exporting via HTTP
+
+In this example, traces will be exported via OTLP/HTTP to the OpenTelemetry Collector.
+
+{{< text bash >}}
+$ cat <<'EOF' > tracing.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    extensionProviders:
+    - name: otel-tracing
+      opentelemetry:
+        port: 4318
+        service: opentelemetry-collector.otel-collector.svc.cluster.local
+        http:
+          path: "v1/traces"
+          timeout: 10s
+          headers:
+          - name: "my-custom-header"
+            value: "some-value"
+EOF
+$ istioctl install -f ./tracing.yaml --skip-confirmation
+kubectl label namespace default istio-injection=enabled
+{{< /text >}}
+
+{{< tip >}}
+You can also send traces directly to your tracing back-end of choice via HTTP, without needing a Collector.
+For that, you will need to first define a [ServiceEntry](/docs/reference/config/networking/service-entry/).
+{{< /tip >}}
+
+## Enable tracing for mesh via Telemetry API
+
+Enable tracing by applying the following configuration:
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: otel-demo
+  namespace: istio-system
+spec:
+  tracing:
+  - providers:
+    - name: otel-tracing
+    randomSamplingPercentage: 100
+    customTags:
+      "my-attribute":
+        literal:
+          value: "default-value"
+EOF
+{{< /text >}}
+
+## Deploy the Bookinfo Application
+
+Deploy the [Bookinfo](/docs/examples/bookinfo/#deploying-the-application) sample application.
+
+## Generating traces using the Bookinfo sample
+
+1.  When the Bookinfo application is up and running, access `http://$GATEWAY_URL/productpage`
+one or more times to generate trace information.
```

Comment:
```suggestion
    one or more times to generate trace information.
```

Reply:
Fixed in [6e77f44](https://github.com/istio/istio.io/pull/14566/commits/6e77f4410d9b032ea3ba8ef05ee2b4ad3401ef37)

Reply:
Cool, thanks.